SPECS/nginx/nginx.spec

```diff
@@ -39,16 +41,34 @@ The nginx-filesystem package contains the basic directory layout
 for the Nginx server including the correct permissions for the
 directories.
 
+%package otel_ngx_module
+License:        Apache-2.0
+Summary:        OpenTelemetry Nginx Module
+BuildRequires:  grpc-devel
+BuildRequires:  opentelemetry-cpp-devel
+BuildRequires:  protobuf-devel
+Requires:       opentelemetry-cpp
+
+%description otel_ngx_module
+The OpenTelemetry module for Nginx
+
 %prep
 %autosetup -p1
 pushd ../
 mkdir nginx-njs
 tar -C nginx-njs -xf %{SOURCE2}
+mkdir otel-cpp-contrib
+tar -C otel-cpp-contrib -xf %{SOURCE3}
+# The following change is a build break in upstream and a PR has been raised to fix it.
```

Comment:
Can we put a comment in the spec linking to the PR?

Reply:
Done